meta/data/user.seed.json

```diff
@@ -1 +1,81 @@
-[{"_id":{"$oid":"5962b898c8cbe418c473a7d5"},"loginName":"jai","password":"test","__v":0},{"_id":{"$oid":"5962b8b2c8cbe418c473a7d6"},"loginName":"lau","password":"test","__v":0},{"_id":{"$oid":"5962b8c0c8cbe418c473a7d7"},"loginName":"ber","password":"test","__v":0}]
+[{
+	"_id": {
+		"$oid": "597e01ba8be16426185a796d"
+	},
+	"email": "joe.doe@foo.com",
+	"name": "Joe",
+	"lastName": "Doe",
+	"avatar": "https://www.google.es/imgres?imgurl=http%3A%2F%2Fcimg.tvgcdn.net%2Fi%2Fr%2F2015%2F09%2F21%2Fb6335653-f456-47b2-a55b-e2f80eebf713%2Fthumbnail%2F1300x867%2Fab98b2214aa92b6eba098522fc79899e%2F150921-news-the-muppets-animal.jpg\u0026imgrefurl=http%3A%2F%2Fwww.tvguide.com%2Fnews%2Fthe-muppets-premiere-abc%2F\u0026docid=X9kDclwlTqpPJM\u0026tbnid=uj9RACmLTFyP6M%3A\u0026vet=10ahUKEwjzy82TnbHVAhUNLVAKHbSHCLUQMwgtKAcwBw..i\u0026w=1300\u0026h=867\u0026bih=736\u0026biw=1540\u0026q=imagenes%20muppets\u0026ved=0ahUKEwjzy82TnbHVAhUNLVAKHbSHCLUQMwgtKAcwBw\u0026iact=mrc\u0026uact=8",
```

Comment:
I just saw the ones being used, they are not local, but well let's use the same so we get similar look & feel (and no isseus with aspect ratio etc).

https://cdn1.iconfinder.com/data/icons/user-pictures/101/malecostume-64.png
https://cdn1.iconfinder.com/data/icons/user-pictures/100/female1-64.png
https://cdn1.iconfinder.com/data/icons/user-pictures/100/boy-64.png

We will refine it in further sprints